add-on/jar.mn

```diff
@@ -118,7 +53,16 @@
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-aero@2x.png          os=WINNT osversion=6.1
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-win8@2x.png         os=WINNT osversion=6.2
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-win8@2x.png         os=WINNT osversion=6.3
-  locale/                        (chrome/locale/*)
+  # Due to the way Loop's L10n works, we include a fallback to en-US.
+  content/locale/en-US/          (chrome/locale/en-US/*)
```

Comment:
To be clear, this /needs/ to be packaged outside of the locale directory because there's checks elsewhere to make sure there's only one packaged locale? Does it make sense to be explicit in the packaging by putting at `content/locale-fallback/`? Otherwise other places that reference `content/locale/en-US` might seem like a misplaced locale.